packages/repository/src/relations/relation.types.ts

```diff
@@ -87,12 +87,12 @@ export interface HasManyThroughDefinition extends RelationDefinitionBase {
   /**
    * The foreign key in the source model, e.g. Customer#id.
    */
-  keyFrom: string;
+  keyFrom?: string;
 
   /**
    * The primary key of the target model, e.g Seller#id.
    */
-  keyTo: string;
+  keyTo?: string;
 
   through: {
```

Comment:
A side note: in the future, I would like to add support for convenience shortcuts, e.g.

```ts
@hasMany(() => Seller, {through: () => Order})
```

See https://github.com/strongloop/loopback-next/pull/2359#discussion_r317012754

This is out of scope of this pull request though.